ops/terraform/modules/resources/templates/fhir_server.tpl

```diff
@@ -20,7 +39,7 @@ aws s3 cp s3://bfd-mgmt-admin-${accountId}/ansible/vault.password .
 
 # The extra_vars.json file from the previous build step contains a few incorrect values
 # and needs to get trimmed down to the following
-cat <<EOF >> extra_vars.json
+cat <<EOF > extra_vars.json
```

Comment:
This never mattered before since user-data only ran once, but now that it can run multiple times, doing an append is bad news!